eo-maven-plugin/src/test/java/org/eolang/maven/PullMojoTest.java

```diff
@@ -90,17 +98,17 @@ void pullsFromProbes(@TempDir final Path temp) throws IOException {
                 "        1337",
                 "        228"
             )
-            .with("objectionary", objectionary)
-            .execute(new FakeMaven.Pull());
-        MatcherAssert.assertThat(
-            new Home(temp.resolve("target")).exists(
-                Paths.get(
-                    String.format(
-                        "%s/org/eolang/io/stdout.eo",
-                        PullMojo.DIR
+            .with(
+                "objectionaries",
+                new Objectionaries.Fake(
+                    new OyRemote(
+                        new ChCompound(null, null, "master")
```

Comment:
@maxonfjvipon Passing null values might be rather dangerous:
https://www.yegor256.com/2014/05/13/why-null-is-bad.html

Reply:
@volodya-lombrozo I didn't touch constructing of this hash, just moved it

Reply:
@volodya-lombrozo and implementation of the class assumes that some of the arguments can be `null`
```java
final CommitHash ret;
if (file == null && hash == null) {
    ret = new ChRemote(tag);
} else if (hash == null) {
     ret = new ChText(file, tag);
} else {
     ret = new ChPattern(hash, tag);
}
return new ChCached(ret);
```

Reply:
@maxonfjvipon Looks like we have to remove that class :) What do you think?

Reply:
@volodya-lombrozo maybe, as I understand it's used only for the test purposes. I can make a todo to think about its fate later